tools/gcp-broker-provider/bin/gcp-broker.sh

```diff
@@ -57,6 +57,12 @@ function configureGCloud() {
     return 0
 }
 
+function deleteGoogleApplicationCredentials () {
```

Comment:
we can make it more general and put at the beginning of the script this:

```bash
trap cleanup EXIT SIGINT SIGTERM

function cleanup() {
    printf  "\n- Cleaning up...\n"

    echo "Ensure gcloud credentials are removed"
    rm "${GOOGLE_APPLICATION_CREDENTIALS}"
    echo "OK"
}
```

for now, you have one specified command for clean-up. To make it more extensible you can just define the clean-up function and as a first step implement cleaning the credentials, in future we can add next steps. 

what do u think?

Reply:
Yeah, good idea 👍